lib/pact/message/consumer/interaction_decorator.rb

```diff
@@ -36,7 +36,9 @@ def decorate_contents
         end
 
         def extract_contents
-          Pact::Reification.from_term(message.contents.contents)
+          message.events.map do |message_content|
+            Pact::Reification.from_term(message_content.contents)
+          end
```

Comment:
📓 It will return an array even for a single record, so when we have only one message content, we show an array `[content]` in Contract, when we have multiple messages, we show an array of arrays `[[Content]]`